app/main.dev.ts

```diff
@@ -123,6 +124,11 @@ app.on( 'ready', async () => {
             if ( commandLine.length >= 2 && uri ) {
                 onReceiveUrl( store, uri );
             }
+
+            if ( commandLine.includes( '--trigger-update' ) ) {
+                // eslint-disable-next-line no-new
+                new AppUpdater( store );
```

Comment:
Do we need a `new` instance? Can the old one be reused?

Reply:
I don't think this would create a new Instance as app Update takes place on the main process only